specification.md

```diff
@@ -1719,28 +1720,28 @@ Configuration with other inputs available to the Agent, e.g. a locally available
 configuration.
 
 Once the Effective Configuration is formed the Agent uses it for its operation
-and will also report the Effective Configuration to the OpAMP Server via the
+and the Client will also report the Effective Configuration to the OpAMP Server via the
 [effective_config](#agenttoservereffective_config) field of status report. The Server
 typically allows the end user to see the effective configuration alongside other
-data reported in the status reported by the Agent.
+data reported in the status reported by the Client.
 
-The Agent MUST set the ReportsEffectiveConfig bit of AgentToServer.capabilities
+The Client MUST set the ReportsEffectiveConfig bit of AgentToServer.capabilities
 if the Agent is capable of reporting effective configuration. If the bit is not
 set the Server should not expect the AgentToServer.effective_config field to be
 set.
 
 Here is the typical configuration sequence diagram:
 
 ```
-               Agent                              Server
+   Agent       Client                             Server
 
-                 │ AgentToServer{}                   │   ┌─────────┐
-                 ├──────────────────────────────────►├──►│ Process │
-                 │                                   │   │ Status  │
+     │           │ AgentToServer{}                   │   ┌─────────┐
+     │           ├──────────────────────────────────►├──►│ Process │
+     │           │                                   │   │ Status  │
 Local     Remote │                                   │   │ and     │
 Config    Config │ ServerToAgent{AgentRemoteConfig}  │   │ Fetch   │
-  │     ┌────────┤◄──────────────────────────────────┤◄──┤ Config  │
-  ▼     ▼        │                                   │   └─────────┘
+  │  │  ┌────────┤◄──────────────────────────────────┤◄──┤ Config  │
+  ▼  │  ▼        │                                   │   └─────────┘
```

Comment:
Can we make the vertical line for the Agent continuous? Probably "Local Config" to the left a bit to make room for the vertical line.